README.md

```diff
@@ -14,3 +14,47 @@ provided by the bot. You will only need to do this once across all repos using o
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+## Join the Community hub to contribute content
+
+1. Go to the **hub** node in the **Community** workspace.
+1. Click on **My hub** and you'll be prompted to sign into GitHub. If you don't have an account, you'll be redirected to GitHub where you can create one. A GitHub account is only required to contribute and share content from the **My hub** page.
```

Comment:
Click on **Your hub**.........and share content from the **Your hub** page.

Reply:
added in commit https://github.com/microsoft/configmgr-hub/pull/11/commits/0ebfdd1ccb14303527e04f2ac12bca41c1d16c77